src/VisualStudio/Core/Def/Implementation/ProjectSystem/AnalyzerConfigDocumentAsSolutionItemHandler.cs

```diff
@@ -0,0 +1,130 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+#nullable enable
+
+using System;
+using System.Collections.Generic;
+using System.Composition;
+using System.Linq;
+using System.Threading;
+using EnvDTE;
+using EnvDTE80;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.Editor.Shared.Utilities;
+using Microsoft.CodeAnalysis.Extensions;
+using Microsoft.CodeAnalysis.Host.Mef;
+using Microsoft.CodeAnalysis.Options;
+using Microsoft.VisualStudio.Shell;
+using Roslyn.Utilities;
+using Task = System.Threading.Tasks.Task;
+
+namespace Microsoft.VisualStudio.LanguageServices.Implementation.ProjectSystem
+{
+    [Export(typeof(AnalyzerConfigDocumentAsSolutionItemHandler)), Shared]
```

Comment:
💭 I think we can omit the type if it's the same type that the attribute is applied to. When I saw the explicit type, I kept trying to figure out where it was different. 😆 

```suggestion
    [Export, Shared]
```

Reply:
Thanks, fixed.